firebase_admin/messaging.py

```diff
@@ -341,16 +341,19 @@ class Aps(object):
         sound: Name of the sound file to be played with the message (optional).
         content_available: A boolean indicating whether to configure a background update
             notification (optional).
+        mutable_content: A boolean indicating whether to have the message be mutable
+            on device before final delivery (optional).
         category: String identifier representing the message type (optional).
         thread_id: An app-specific string identifier for grouping messages (optional).
     """
 
-    def __init__(self, alert=None, badge=None, sound=None, content_available=None, category=None,
-                 thread_id=None):
+    def __init__(self, alert=None, badge=None, sound=None, content_available=None,
+                 mutable_content=None, category=None, thread_id=None):
```

Comment:
`mutable-content` does not seem to be a standard parameter APNS supports but one of the many possible extensions. I think the right way to support this type of options is by adding a `**kwargs` parameter to the `Aps.__init__()` constructor (or may be a regular dict parameter, so we don't have to deal with mapping underscores to hyphens). In any case, this is an API change, and will need to go through our internal review process. Please hang tight.

Reply:
I had thought of that as a good option as well, I'm happy to swap to that if you'd like, just let me know.  I'll tell you I chose it as is because I think it will become official, it is just newish and I think the doc reference hasn't been updated.